content/en/cosign/overview.md

```diff
@@ -63,23 +63,71 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
 
 To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-### Working with containers
+## Working with containers
 
-Signing and verifying a container is similar to working with blobs. The Cosign command to sign a container image is:
+> NOTE: You will need access to a container registry for cosign to work with. [ttl.sh](https://ttl.sh/) offers free, short-lived (i.e., hours), anonymous container image hosting if you just want to try these commands out.
+
+Using `ttl.sh` and [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to prepare the image that we want to sign. Run the following:
+
+```
+$ SRC_IMAGE=busybox
+$ SRC_DIGEST=$(crane digest busybox)
+$ IMAGE_URI=ttl.sh/$(uuidgen | head -c 8 | tr 'A-Z' 'a-z')
+$ crane cp $SRC_IMAGE@$SRC_DIGEST $IMAGE_URI:1h
+$ IMAGE_URI_DIGEST=$IMAGE_URI@$SRC_DIGEST
+```
+
+### Keyless signing of container
 
 ```
-$ cosign sign <image URI>
+$ cosign sign $IMAGE_URI_DIGEST
+Generating ephemeral keys...
+Retrieving signed certificate...
+Successfully verified SCT...
+
+	Note that there may be personally identifiable information associated with this signed artifact.
+	This may include the email address associated with the account with which you authenticate.
+	This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.
+
+By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs.
+Are you sure you would like to continue? [y/N] y
+tlog entry created with index: 18914440
+Pushing signature to: ttl.sh/4d6d55ae
 ```
 
-This works the same as signing a blob, but the signature and certificate are attached as container metadata.
+### Keyless verifying of container
 
-To verify a signed container image, use the following command:
+This works the same as verifying a blob, but the signature and certificate are attached as container metadata, so there is no need to place the certificate and signature on the verify command. To verify a signed container image, use the following command:
 
 ```
 $ cosign verify <image URI> --certificate-identity=name@example.com
                             --certificate-oidc-issuer=https://accounts.example.com
 ```
 
+> Note that in the following example we use the `regexp` versions of the identity options:
+
+```
+cosign verify $IMAGE_URI_DIGEST --certificate-identity-regexp=.* --certificate-oidc-issuer-regexp=.*
+
+Verification for ttl.sh/4d6d55ae@sha256:b5d6fe0712636ceb7430189de28819e195e8966372edfc2d9409d79402a0dc16 --
```

Comment:
Same comment here, I think we should revert back to a generic example.

Reply:
keeping the ttl.sh example.